app/presenters/single_content_item_presenter.rb

```diff
@@ -146,11 +146,11 @@ def useful_yes_no_total
   end
 
   def on_page_search_rate
-    metric_value = self.total_searches
-    secondary_metric_value = self.total_pviews
+    metric_value_as_float = self.total_searches.delete(',').to_f
```

Comment:
How about using `@metrics['searches'][:value]` here, to avoid the formatting in the first place?

Reply:
@KyleMacPherson @cbaines Sorry I just merged and I didn't notice your comment. 

@KyleMacPherson could you please address @cbaines in a separate PR?

Reply:
@cbaines Nice!